charts/dependentapi-simple-operator/Chart.yaml

```diff
@@ -27,4 +28,4 @@ version: 0.2.4
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: 0.2.3"
\ No newline at end of file
+appVersion: v1beta4
```

Comment:
Is `v1beta4` correct here, or should it be `0.3.0`?

Reply:
In most of the other charts, the appVersion is the latest component spec that it supports (this rule is not applied everywhere - it is in 9 of the 12 charts). I'm happy to discuss what the right answer should be!

![image](https://github.com/user-attachments/assets/49abd170-dd23-49d5-b2c8-90c900cb1e98)